docs/CSBR.md

```diff
@@ -404,14 +931,330 @@ copyright: |
+
+d. `basicConstraints`
+
+   This extension MUST be present and MUST be marked critical. The `cA` field MUST be set true. The `pathLenConstraint` field MAY be present.
+
+e. `keyUsage`
+
+   This extension MUST be present and MUST be marked critical. Bit positions for `keyCertSign` and `cRLSign` MUST be set. If the Subordinate CA Private Key is used for signing OCSP responses, then the `digitalSignature` bit MUST be set.
+
+g. `extKeyUsage`
+
+This extension MUST be present and SHOULD NOT be marked critical.
+
+If the Subordinate CA will be used to issue Code Signing Certificates:
+* `id-kp-codeSigning` MUST be present.
+
+If the Subordinate CA will be used to issue Timestamp Certificates:
+* `id-kp-timeStamping` MUST be present.
+
+Other values MUST NOT be present unless the CA has a business agreement with a Platform vendor requiring that EKU in order to issue a Platform-specific code signing certificate with that EKU.
+
+h. `authorityKeyIdentifier`
+
+   This extension MUST be present and MUST NOT be marked critical.
+
+#### 7.1.2.3 Code signing and Timestamp Certificate
+
+a. `certificatePolicies`
+
+   This extension MUST be present and SHOULD NOT be marked critical.
+
+   * `certificatePolicies:policyIdentifier` (Required)
+
+      A Policy Identifier, defined by the issuing CA, that indicates a Certificate Policy asserting the issuing CA's adherence to and compliance with these Requirements.
+
+   The following fields MAY be present:
+
+   * `certificatePolicies:policyQualifiers:policyQualifierId` (Recommended)
+
+      `id-qt 1` [RFC 5280].
+
+   * `certificatePolicies:policyQualifiers:qualifier:cPSuri` (Optional)
+
+      HTTP URL for the Subordinate CA's Certification Practice Statement, Relying Party Agreement or other pointer to online information provided by the CA.
+
+b. `cRLDistributionPoints`
+
+   This extension MUST be present. It MUST NOT be marked critical, and it MUST contain the HTTP URL of the CA's CRL service.
+
+c. `authorityInformationAccess`
+
+   This extension MUST be present. It MUST NOT be marked critical.
+
+   It MUST contain the HTTP URL of the Issuing CA's certificate (`accessMethod` = 1.3.6.1.5.5.7.48.2).
+   If the CA provides OCSP responses, it MUST contain the HTTP URL of the Issuing CA's OCSP responder (`accessMethod` = 1.3.6.1.5.5.7.48.1).
+    
+d. `basicConstraints` (optional)
+
+   The `cA` field MUST NOT be true.
+
+e. `keyUsage`
+
+   This extension MUST be present and MUST be marked critical.
+
+   The bit position for `digitalSignature` MUST be set. Bit positions for `keyCertSign` and `cRLSign` MUST NOT be set. All other bit positions SHOULD NOT be set.
+
+f. `extKeyUsage`
+
+   If the Certificate is a Code Signing Certificate, then `id-kp-codeSigning` MUST be present and the Lifetime Signing OID (`1.3.6.1.4.1.311.10.3.13`) MAY be present.
+
+   If the Certificate is a timestamp Certificate, then `id-kp-timeStamping` MUST be present and MUST be marked critical.

```

Comment:
If the Certificate is a timestamp Certificate, then:

* this extension MUST be marked critical;
* `id-kp-timeStamping` (1.3.6.1.5.5.7.3.8) MUST be present;
* `anyExtendedKeyUsage` (2.5.29.37.0) MUST NOT be present.